mswindows/osgeo4w/mklibs.sh

```diff
@@ -1,31 +1,14 @@
 #!/bin/sh
 
 set -e
+set -x
 
-if [ -z "$OSGEO4W_POSTFIX" ]; then
-    OSGEO4W_POSTFIX=""
-fi
+[ -d mswindows/osgeo4w/vc ] || mkdir mswindows/osgeo4w/vc
 
-if [ "$OSGEO4W_POSTFIX" != "64" ]; then
-    MACHINE=x86
-else
-    MACHINE=x64
+if [ -n "$VCPATH" ]; then
```

Comment:
do we define $VCPATH anywhere?

Reply:
No, it must be defined when running the script. Similarly to `OSGEO4W_ROOT_MSYS`. E.g.

```
OSGEO4W_ROOT_MSYS=/c/osgeo4w  VCPATH=$PROGRAMFILES/Microsoft Visual Studio/2017/Community ./mswindows/osgeo4w/package.sh
```